clusterloader2/testing/load/rc.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: {{.Name}}
+  labels:
+    kind: loadRC
+spec:
+  replicas: {{RandIntRange .ReplicasMin .ReplicasMax}}
+  selector:
+    name: {{.Name}}
+  template:
+    metadata:
+      labels:
+        name: {{.Name}}
+    spec:
+      containers:
+      - image: k8s.gcr.io/pause:3.1
+        imagePullPolicy: IfNotPresent
```

Comment:
This is a [default](https://kubernetes.io/docs/concepts/containers/images/#updating-images), please remove it. Also from density test's yaml.

Reply:
Done.